scripts/plotting/postfitPlots.py

```diff
@@ -50,11 +52,12 @@
 
 fittype = "prefit" if args.prefit else "postfit"
 ratio = not args.noRatio
+diff = args.logTransform
```

Comment:
Should this work in conjunction with flag "ratio" when one doesn't want the ratio panel ? Maybe --noRatio should become --noRatioDiffPanel or --noBottomPanel, I don't know what's best. It can be dealt with in another PR, not worth running CI only for it

Reply:
Done